### PR TITLE
Added weekly meeting notes, 2017-09-21

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
I may not have got all of the relevant discussion points during the EB part of the meeting, so if you remember something that's missed please say. @jouvin thanks for helping with the live notes.

### CODE REVIEW

File: organization/_posts/2017-09-21-startup.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+title: "HSF Weekly Meeting #109, September 21, 2017"
+layout: default
+---
+
+# HSF Weekly Meeting #109, September 21, 2017
+
+#### *Present*: Ian Bird, Michel Jouvin, Dario Menasce, Pere Mato, Liz Sexton-Kennedy, John Harvey, Benedikt Hegner, Marko Petric, Graeme Stewart, Andre Sailer, Daniel Elvira, David Lange, Mike Sokoloff, Pete Elmer, David Lange, Marco Brako, Thomas Kuhr
```
Comment (hegner) [inline organization/_posts/2017-09-21-startup.md]: Marco Brako was a person connected remotely?

File: organization/_posts/2017-09-21-startup.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+-   #### General matters and Editorial Board
+    - See [slides](https://indico.cern.ch/event/667285/contributions/2727738/attachments/1526808/2389640/CWP_Editorial_Board_2017-09-21.pdf)
+    - Editorial Board: trying to address the LHCC comments about the need to involve more people outside the LHC community (despite the fact that LHCC is only concerned with the LHC) and too much CERN+US dominated. Contacts established with Belle II, LC community, IF, ... 
+    - The Reconstruction and Simulation summaries are very good, so we recommend the structure **Challenges + Current Approaches + Roadmap**
+    - It is better to organise R&D actions on 2 time scales: 2y and 5y (rather than 1/3/5). Doesn’t mean that all actions will be delayed by 2 years of course.
+    - Effort description: minimum granularity is whether it is an additional effort to what we are doing or just re-assigning existing effort (e.g., easy things with modest impact should be done)
+    - Synthesis document
+        - A small team set up to prepare a first draft that will be discussed by EB and community
+        - Keep sections 6 and 7 separate for now
```
Comment (hegner) [inline organization/_posts/2017-09-21-startup.md]: (see slide 5 of the presentation)

File: organization/_posts/2017-09-21-startup.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+-   #### Facilities, and Distributed Computing
+    - Ian dealing with comments now. Executive summary outline available. Roadmap for R&D activities needs to be done/finalized: Ian discussing with various people about it.
+    - Reviewed by Belle II (Thomas)
+
+-   #### Conditions Access
+    - Document ready for review.
+
+-   #### Generators
+    - Graeme spoke to Taylor who is continuing to gather input. Torben Feber said he could provide some input from Belle II.
```
Comment (hegner) [inline organization/_posts/2017-09-21-startup.md]: Torben Ferber

File: organization/_posts/2017-09-21-startup.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    - No news, Grame will follow up.
+
+-   #### Software Development
+    - Updated document in line with some comments received. Revamping the roadmap section. Should finish tomorrow.
+
+-   #### Software Trigger and Event Reconstruction
+    - No updates, ATLAS were asked to give feedback.
+
+-   #### Data Access, Organisation and Management
+    - Meeting scheduled right after our weekly meeting today: more after it.
```
Comment (jouvin) [inline organization/_posts/2017-09-21-startup.md]: I forgot to update the live notes after our meeting... May be you want to update the status to 
* Contents more or less ok but some work in progress to better define the challenges (currently dispersed in the text) and write a proper R&D Roadmap (currently an unordered, unprioritized bullet list)
* Progress expected at next WG meeting (just after the HSF weekly meeting)

File: organization/_posts/2017-09-21-startup.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+### Licensing
+- Presentation by Graeme and discussion within ATLAS during their workshop. No strong opinion on which license to use. Important is to understand the boundary conditions
+    - Policy of US national labs
+    - Requirements of industrial partners
+- Discussion with CMS about the same topics
+- Mixed licensing still looks like a good option in case openlab-cooperation requires us to provide non-GPL access to code
+- Belle II: licensing TF should meet Oct. 12, at the occasion of the Belle II Collaboration Week
+
+### Training
+- Dario asked Manuel Giffels (contact person for the GridKa School of Computing) to provide me feedback on their experience with WikiToLearn (for eventual future improvements). Waiting for answers.
```
Comment (hegner) [inline organization/_posts/2017-09-21-startup.md]: remove "me"

File: organization/_posts/2017-09-21-startup.md (hunk 1 above)
Comment (graeme-a-stewart) [reply organization/_posts/2017-09-21-startup.md]: Yes - I grabbed his name from Vidyo. He's in Belle II.

File: organization/_posts/2017-09-21-startup.md (hunk 2 above)
Comment (graeme-a-stewart) [reply organization/_posts/2017-09-21-startup.md]: Done

File: organization/_posts/2017-09-21-startup.md (hunk 3 above)
Comment (graeme-a-stewart) [reply organization/_posts/2017-09-21-startup.md]: Done

**Comment by graeme-a-stewart** [comment]
@jouvin - thanks for the Data Access news, I added it.

**Pull request merged**